get-resource.sh

```diff
@@ -63,6 +63,10 @@ else
       unxz "$RHCOS_IMAGE_FILENAME_RAW"
     fi
 
+    if [ -n "$IP_OPTIONS" ] ; then
+        LIBGUESTFS_BACKEND=direct virt-edit -a "$RHCOS_IMAGE_FILENAME_QCOW" -m /dev/sda3 /boot/loader/entries/ostree-1-rhcos.conf -e "s/^options/options ${IP_OPTIONS}/"
```

Comment:
Hi, when testing the workaround we noticed that in OCP 4.6 for example the boot disk is sda1, so there might be as issue with the hardcoded /dev/sda3 if this fix will be backported.
The right boot disk can be found with virt-filesystems, for example:
BOOT_DISK=$(virt-filesystems -a $IMAGE -l | grep boot | cut -f1 -d" ")

Reply:
Thanks for the great one-liner!